packages/xchain-ethereum/src/utils.ts

```diff
@@ -1,12 +1,33 @@
 import { Fees, Network as XChainNetwork, Tx } from '@xchainjs/xchain-client'
-import { Asset, AssetETH, assetFromString, baseAmount, ETHChain } from '@xchainjs/xchain-util'
-import { Network as EthNetwork, Address, ETHTransactionInfo, TokenTransactionInfo } from './types'
-import { ethers } from 'ethers'
+import {
+  Asset,
+  AssetETH,
+  assetFromString,
+  baseAmount,
+  ETHChain,
+  BaseAmount,
+  assetToString,
+} from '@xchainjs/xchain-util'
+import {
+  Network as EthNetwork,
+  Address,
+  ETHTransactionInfo,
+  TokenTransactionInfo,
+  FeesWithGasPricesAndLimits,
+} from './types'
+import { ethers, BigNumber } from 'ethers'
+import { parseUnits } from 'ethers/lib/utils'
 
 export const ETH_DECIMAL = 18
-export const DEFAULT_GASLIMIT = 63000
 export const ETHPLORER_FREEKEY = 'freekey'
 
+// from https://github.com/MetaMask/metamask-extension/blob/ee205b893fe61dc4736efc576e0663189a9d23da/ui/app/pages/send/send.constants.js#L39
```

Comment:
[nitpick] ... and based on recommendations of https://ethgasstation.info/blog/gas-limit/